go/signedexchange/cmd/gen-certurl/main.go

```diff
@@ -58,9 +63,13 @@ func run(pemFilePath, ocspFilePath, sctDirPath string) error {
 		if err != nil {
 			return err
 		}
+	} else {
+		if !certurl.HasEmbeddedSCT(certs[0], parsed_ocsp) {
+			log.Println("Warning: Neither cert nor OCSP have embedded SCT list. Use -sctDir flag to add SCT from files.")
```

Comment:
For warnings, I think `fmt.Fprintf(os.Stderr)` should be fine.

Example usage in Go command: https://golang.org/src/cmd/go/main.go

Reply:
Done